lib/app/constants/bottom_navigation_items.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:gruene_app/app/constants/routes.dart';
+
+class BottomNavigationItem {
```

Comment:
:wrench: I'd prefer a more general naming, e.g. `NavigationItem`. I think the fact that it is used in a bottom navigation bar is an implementation detail not really relevant here. As mentioned below, there is already a successor component `NavigationBar` which we might use in the future, too (or some other kind of navigation).

Reply:
Would be okay for me, but I think we will need other navigation items in the app which have other properties. So I decided to use this naming to know where this class is used.